app/controllers/provider/admin/backend_apis/metrics_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApis::MetricsController < Provider::Admin::BackendApis::BaseController
+  before_action :find_metric, except: %i[new create index]
+  before_action :build_metric, only: %i[new create]
+
+  activate_menu :backend_api, :methods_metrics
+  sublayout 'api/service'
+
+  def create
+    # TODO
+  end
+
+  def index
+    @metrics = @backend_api.metrics.top_level.includes(:proxy_rules)
+    @service = @backend_api.first_service # FIXME: This is needed because the page is still using a helper shared with the old metrics page (app/helpers/services_helper.rb:26)
```

Comment:
If the plan is to change this soon, then ok, but otherwise I would suggest at least having this in a separate method with a `before_action` because then:
1. We don't repeat this line at the beginning of so many methods.
2. It is clearer (visually, as soon as we open the file) that it is a dependency and absolutely needed.

Reply:
No need. This too will change soon as part of https://issues.jboss.org/browse/THREESCALE-3089.

Reply:
At least the ones in `#new` and `#edit` we'll get rid soon as part of https://issues.jboss.org/browse/THREESCALE-3089. Let's set a reminder for this one to be fixed as well.